collivery.php

```diff
@@ -3,19 +3,19 @@
 use MdsSupportingClasses\MdsColliveryService;
 
 define('_MDS_DIR_', __DIR__);
-define('MDS_VERSION', '3.7.4');
+define('MDS_VERSION', '3.8.0');
 include 'autoload.php';
 require_once ABSPATH.'wp-includes/functions.php';
 
 /*
  * Plugin Name: MDS Collivery
  * Plugin URI: https://collivery.net/integration/woocommerce
  * Description: Plugin to add support for MDS Collivery in WooCommerce.
- * Version: 3.7.4
+ * Version: 3.8.0
  * Author: MDS Technologies
  * License: GNU/GPL version 3 or later: http://www.gnu.org/licenses/gpl.html
  * WC requires at least: 3.5
```

Comment:
Does this still work on WC 3.5?

Reply:
I'm not sure, we've not tested it on WC 3.5.

Reply:
We've been testing on WordPress 5.4.2 and Woocommerce 4.5.2.

Should I just update that value to 4.5.2?

Reply:
@WilRyan @MarnuLombard ~ Just drawing attention to this.